src/Entity/Ern41/NewReleaseMessage.php

```diff
@@ -41,7 +41,7 @@ class NewReleaseMessage
     /**
      * A Composite containing details of one or more Parties relating to the reported MusicalWorks.
      *
-     * @var \DedexBundle\Entity\Ern41\PartyType[] $partyList
+     * @var \DedexBundle\Entity\Ern41\PartyListType[] $partyList
```

Comment:
this shouldn't by an array any longer (same for dealList)
```suggestion
     * @var \DedexBundle\Entity\Ern41\PartyListType $partyList
```